### ISSUE TITLE
monitor.c(176,108): warning C4295: 'ssname': array is too small to include a terminating null character

### ISSUE THREAD

**Issue by Blonder** [pete-gordon/oricutron#144]
The size of SSNAME_LEN is (SNAME_LEN-3) and SNAME_LEN = 11 so SSNAME_LEN = 8,
but some strings are larger than 8 e.g. BankedArea that is 10 characters. If I add +2 to SSNAME_LEN all is fine.

**Comment by assinie** [comment]
"BankedArea" is the sname, the ssname is "BankedA\x16" which is 8 characters

**Comment by assinie** [comment]
Maybe it's better to change the string instead of changing the size of ssname otherwise the display in the monitor may be badly formatted

**Comment by Blonder** [comment]
> "BankedArea" is the sname, the ssname is "BankedA\x16" which is 8 characters

Yes you are right, the Visual Studio Compiler complains about ssname:
`warning C4295: 'ssname': array is too small to include a terminating null character`

**Comment by assinie** [comment]
Found: 

- "VIA_IORA2"
- "VIA2_IORB"
- "VIA2_IORA"
- "VIA2_DDRA"
- "VIA2_DDRB"

**Comment by Blonder** [comment]
When these names are correctly shown in the monitor we can increase the buf size.

**Comment by assinie** [comment]
The ssname is only used when disassembling zero page indexed instructions.

If I add 2 to SSNAME_LEN, the 'd' command display is correct:
![image](https://user-images.githubusercontent.com/15648613/68285911-72334d00-0080-11ea-8989-d3a0b80bb3ae.png)

 but the 'a' command isn't (there is a carriage return before the last character of the line):
![image](https://user-images.githubusercontent.com/15648613/68285816-41ebae80-0080-11ea-9643-0b7351311efa.png)

I think the mon_oprintf() function needs some modifications too.

**Comment by assinie** [comment]
Ok, after changes to the mon_oprintf () function, the display is now correct with the 'a' command too:
![image](https://user-images.githubusercontent.com/15648613/68286495-77dd6280-0081-11ea-994e-26cc11b17b1d.png)

**Comment by assinie** [comment]
I just realized that the disassembler part of the "6502 Status" window must also be modified (1 character less than the "Monitor" window):
![image](https://user-images.githubusercontent.com/15648613/68378322-2fd74200-014c-11ea-92bb-66d9ec399216.png)

**Comment by Blonder** [comment]
I think it's better to shorten the texts that are too long, that has lesser impact.

**Comment by assinie** [comment]
Yes, I agree
